cypress/integration/e2e-setClusterDnsDomain.spec.js

```diff
@@ -0,0 +1,21 @@
+import {
+  CLUSTER_NAME,
+  openCluster,
+  saveClusterDetails,
+  setClusterDnsDomain,
+  DNS_DOMAIN_NAME,
+} from './shared';
+
+describe('Set DNS Domain -> ', () => {
+  it('Open the cluster details', () => {
+    openCluster(CLUSTER_NAME);
+  });
+
+  it('Set DNS Domain', () => {
+    setClusterDnsDomain(DNS_DOMAIN_NAME);
+  });
+
+  it('Save changes', () => {
+    saveClusterDetails(cy);
```

Comment:
Might be good to check list of errors (either expected ones or none displayed once it is clear that response from API is received).

Reply:
As discussed the list is there only on error and not on happy flow